homeassistant/components/light/zwave.py

```diff
@@ -88,14 +88,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     value.set_change_verified(False)
 
-    if node.has_command_class(zwave.const.COMMAND_CLASS_SWITCH_COLOR):
-        try:
-            add_devices([ZwaveColorLight(value)])
-        except ValueError as exception:
-            _LOGGER.warning(
-                "Error initializing as color bulb: %s "
-                "Initializing as standard dimmer.", exception)
-            add_devices([ZwaveDimmer(value)])
```

Comment:
That's correct. If the node supports the color command class, it's initialized as a color bulb. This removes the scenario where the color bulb would be initialized as a standard dimmer, since we're now asynchronously waiting for the color values if they aren't there in the first load.

Today RGB bulbs are added as one entity that manages both values, but it is only initialized correctly if both values are present at initialization.